### DISCUSSION TITLE
Images with fill/cover end up looking fried

### DISCUSSION THREAD

**Discussion by zanewolf** [vercel/next.js#43004] [Help]
I am attempting to make profile cards. When I use fill/contain, the images look great but are not filling the circular div: 
![image](https://user-images.githubusercontent.com/30910417/202250368-1e2c927a-1132-471a-99e1-e6cb34589427.png)

 
And when I use fill/cover, they are the circular profile pics I want but the images themselves look a bit fried. Adding the unoptimized parameter did not help. 
![image](https://user-images.githubusercontent.com/30910417/202250292-80e74846-93a4-45ee-a14f-2551f46231dd.png)

Any ideas would be greatly appreciated!

`<div className="imageContainer m-auto block w-[40%] relative h-full mt-2 rounded-full overflow-hidden">
                        {image ?
                                <Image
                                    loader={myLoader}
                                    className={'profileImage'}
                                    src={profileImage}
                                    alt={'Profile picture of ' + profile.fields.Name}
                                    layout={'fill'}
                                    objectFit={'contain'}
                                    unoptimized={false}
                                    quality={100}
                                    loading={"eager"}
                                    priority = {index < 10 ? true : false}
                                />
                            :
                            <div className="avatarWrapper ">
                                <Avatar
                                    size={'12vh'}
                                    name={profile.fields.Name}
                                    variant="beam"
                                    colors={colors}
                                />
                            </div>
                        }
                    </div>`

**Comment by zanewolf** [comment]
bump. Any help please?

**Comment by zanewolf** [comment]
Using 'contain' with style={{width: "100%", height: "auto"}} doesn't work.

**Reply by kerns** [reply]
This is surely not helpful but I don't think the image looks great in either scenario. You just notice the artifacts more in the larger / stretched circular version. Are you sure the source image is good? Have you tried not using a % width on the image container?

**Reply by zanewolf** [reply]
I tried using pixel values, set a perfect circle, and removed the fill property, setting the width and height to 500. It now looks good for all portrait-oriented photos. However, for landscape-oriented photos, they are cut-off. 

![image](https://user-images.githubusercontent.com/30910417/206914597-5cd0101b-9c36-44ac-a79b-a87e8f0f5109.png)

Adding ```style={{minWidth:"100px",width:"auto",height:"auto",minHeight:"100px"}}``` causes the landscape-portraits to warp. 

![image](https://user-images.githubusercontent.com/30910417/206914710-9f7a7cdc-1151-4322-b94c-820fe688a4d6.png)
